crates/scanner/src/cache.rs

```diff
@@ -13,6 +14,9 @@ pub struct TagCache {
 
     pub version: u32,
 
+    /// Combined hash of embedded and local wordlists
+    pub wordlist_hash: u32,
+
```

Comment:
When you change the tagcache structure, make sure you bump the tagcache version as well